### ISSUE TITLE
[Enterprise Fee Summary] Hide report link with feature flag

### ISSUE THREAD

**Issue by kristinalim** [openfoodfoundation/openfoodnetwork#3332]
Hide report link with feature flag.

## Description

This was recommended by @sauloperez [in Slack](https://openfoodnetwork.slack.com/messages/C2GQ45KNU/convo/C2GQ45KNU-1547186925.158100/).

The link should be visible only to superadmins. However, any enterprise user can also access the report by going directly to the URL.

## Expected Behavior

A non-superadmin user should not see "Enterprise Fee Summary" in the list of reports.

## Actual Behavior

All enterprise users see "Enterprise Fee Summary" in the list of reports.

## Steps to Reproduce

In the admin section:

1. Log in as a non-superadmin user.
2. Go to the admin section.
3. Click on "Reports".
4. Notice the "Enterprise Fee Summary" link.

**Comment by kristinalim** [comment]
**UPDATE:** I changed the issue title and description

**Comment by kirstenalarsen** [comment]
I disagree - why would we only let super-admin see this, it would completely undermine the purpose of the report which is Hubs being able to use fees and easily see how much is owing to whom. I say delete this. Not needed or wanted at all

**Comment by tschumilas** [comment]
I also disagree -  the whole reason we initiated the fee report was so that enterprise managers could see the break down of various fees.  How else can a OC coordinator know how much to pay to various fee setters if they don't see the report?

**Comment by kristinalim** [comment]
Sorry, I should have clarified this in the issue description. This is only done through a temporary feature flag, which would be removed as soon as the report has been fully tested by superadmins in production. Removal of the feature flag would then make this feature available to all users.

Quoting what @mkllnk said in Slack here:

> The issue about making it only available for superadmins was created to release it early while it's not ready. If you now say it's ready for the public, we don't need that issue any more.

We can close if not needed.
